nsxt/removed_resource_wrapper.go

```diff
@@ -66,15 +66,18 @@ func deleteWrapper(originalFunc schema.DeleteFunc, name string) schema.DeleteFun
 }
 
 func importerWrapper(originalImporter *schema.ResourceImporter, name string) *schema.ResourceImporter {
+	newImporter := new(schema.ResourceImporter)
```

Comment:
would it be more straightforward to return `originalImporter` here, and let the `Read` fail?

Reply:
that was the previous code.
Probably you meant do not wrapping the importer at all and let import fail at read?